core/modules/dashboard/config/layout.layout.dashboard.json

```diff
@@ -0,0 +1,298 @@
+                    "block_settings": [],
+                    "contexts": []
+                },
+                "uuid": "eb936b48-d2ec-4d9f-9dd9-ef24a1ad3493",
+                "style": {
+                    "plugin": "default",
+                    "data": {
+                        "settings": {
+                            "classes": ""
+                        }
+                    }
+                }
+            }
+        },
+        "a144d9d3-0cac-4c20-989e-75220ab686cb": {
+            "plugin": "layout:custom_block",
+            "data": {
+                "module": "layout",
+                "delta": "custom_block",
+                "settings": {
+                    "title_display": "default",
+                    "title": "",
+                    "style": "default",
+                    "block_settings": [],
+                    "contexts": [],
+                    "content": "<p>An overview of your website.</p>\r\n",
+                    "format": "filtered_html"
+                },
+                "uuid": "a144d9d3-0cac-4c20-989e-75220ab686cb",
+                "style": {
+                    "plugin": "default",
+                    "data": {
+                        "settings": {
+                            "classes": ""
+                        }
+                    }
+                }
+            }
+        },
+        "aab75ee1-7a16-4137-b253-83a0f6323936": {
+            "plugin": "dashboard:updates",
+            "data": {
+                "module": "dashboard",
+                "delta": "updates",
+                "settings": {
+                    "title_display": "default",
+                    "title": "",
+                    "style": "default",
+                    "block_settings": [],
+                    "contexts": [],
+                    "display": {
+                        "core": "core",
+                        "module": "module",
+                        "theme": "theme",
+                        "layout": "layout"
+                    }
+                },
+                "uuid": "aab75ee1-7a16-4137-b253-83a0f6323936",
+                "style": {
+                    "plugin": "default",
+                    "data": {
+                        "settings": {
+                            "classes": ""
+                        }
+                    }
+                }
+            }
+        },
+        "5a4ec847-16c1-45dd-b168-58afed5081d4": {
+            "plugin": "dashboard:create",
+            "data": {
+                "module": "dashboard",
+                "delta": "create",
+                "settings": {
+                    "title_display": "default",
+                    "title": "",
+                    "style": "default",
+                    "block_settings": [],
+                    "contexts": [],
+                    "types": {
+                        "page": "page",
+                        "post": "post",
+                        "blog_post": "blog_post"
```

Comment:
...was this a custom content type on your local perhaps? Shouldn't be here.

Reply:
Yeah, seems as though all types should be omitted with the behavior that if none are selected, all are available?

Reply:
fixed in latest PR.